helix-core/src/main/java/org/apache/helix/monitoring/mbeans/MessageQueueMonitor.java

```diff
@@ -70,9 +72,9 @@ public void init() {
    * Remove this bean from the server
    */
   public void reset() {
-    _messageQueueBacklog = 0;
+    _messageQueueBacklog.updateValue(0L);
     try {
-      unregister(getObjectName(getBeanName()));
+      unregister();
     } catch (Exception e) {
       LOG.error("Fail to register MessageQueueMonitor", e);
```

Comment:
NIT: Am I missed something in another PR? I did not find the unregister() function.

Reply:
It's defined in the parent class (DynamicMBeanProvider).